lib/Traits/WebMapServiceCatalogItemTraits.ts

```diff
@@ -232,10 +234,10 @@ export default class WebMapServiceCatalogItemTraits extends mixTraits(
 
   @primitiveTrait({
     type: "boolean",
-    name: "Disable style selector",
-    description: "When true, disables the style selector in the workbench"
+    name: "Disable dimension selectors",
+    description: "Disables the dimension selectors when true"
```

Comment:
Still keep the "in the workbench" wording?
The way I see it, if I was using the catalog editor and reading this description, and didn't know what a dimension selector was, it would be confusing. So at least some hint that it's workbench related would be helpful